configs/a64fx_SME.yaml

```diff
@@ -154,11 +137,6 @@ Reservation-Stations:
     Dispatch-Rate: 2
```

Comment:
What does dispatch rate represent? If it is how many instructions can be issued per cycle, why for RSs with only one connected EU is is more than 1?

Reply:
It's what you said. Won't effect anything due to the notion of ports being busy but I'll change it for clarity